Mage.Sets/src/mage/sets/Invasion.java

```diff
@@ -375,4 +383,118 @@ private Invasion() {
+    private final CardRun commonD = new CardRun(true, "112", "248", "147", "247", "205", "286", "7", "155", "290", "187", "264", "22", "271", "67", "254", "120", "260", "148", "80", "289", "117", "11", "229", "70", "286", "120", "248", "148", "289", "187", "112", "11", "260", "80", "188", "290", "22", "247", "70", "205", "264", "7", "147", "254", "188", "117", "271", "155", "67", "229");
+    private final CardRun uncommon = new CardRun(false, "91", "297", "228", "2", "94", "320", "295", "233", "234", "5", "6", "9", "298", "138", "184", "238", "140", "239", "299", "240", "321", "96", "300", "301", "99", "51", "302", "103", "303", "322", "185", "55", "57", "143", "246", "145", "107", "146", "108", "250", "251", "252", "191", "150", "21", "152", "255", "60", "62", "198", "258", "294", "119", "24", "309", "202", "159", "69", "121", "265", "32", "266", "310", "207", "35", "326", "269", "36", "210", "165", "311", "270", "166", "167", "327", "272", "168", "74", "273", "274", "275", "312", "39", "293", "292", "278", "212", "77", "213", "43", "314", "81", "218", "127", "315", "316", "283", "131", "133", "330", "221", "284", "84", "288", "223", "87", "296", "44", "291", "90");
+    // omitted Crystal Spray "50" - unimplemented
+    private final CardRun rare = new CardRun(false, "226", "227", "1", "93", "230", "231", "4", "232", "235", "137", "182", "236", "47", "183", "48", "139", "237", "241", "141", "49", "242", "12", "97", "243", "13", "100", "101", "52", "15", "102", "244", "245", "186", "54", "16", "144", "18", "249", "19", "190", "304", "253", "193", "149", "194", "325", "153", "305", "154", "59", "110", "61", "256", "199", "113", "257", "156", "259", "306", "115", "116", "307", "308", "68", "27", "261", "262", "263", "206", "33", "267", "34", "161", "268", "71", "208", "209", "169", "276", "38", "125", "277", "171", "76", "42", "173", "279", "78", "313", "79", "214", "216", "175", "280", "128", "129", "317", "281", "130", "282", "318", "178", "219", "220", "285", "287", "88", "45", "135");
+
+    // either A then B, or B then A
+    private final BoosterStructure AAABBB = new BoosterStructure(
+            commonA, commonA, commonA,
+            commonB, commonB, commonB
+    );
+    private final BoosterStructure BBBAAA = new BoosterStructure(
+            commonB, commonB, commonB,
+            commonA, commonA, commonA
+    );
+    private final BoosterStructure AAAABB = new BoosterStructure(
+            commonA, commonA, commonA, commonA,
+            commonB, commonB
+    );
+    private final BoosterStructure BBBBAA = new BoosterStructure(
+            commonB, commonB, commonB, commonB,
+            commonA, commonA
+    );
+    private final BoosterStructure BBAAA = new BoosterStructure(
```

Comment:
Nope, you still need to check all usages to make sure you have the right number of cards.